src/crawlers/cheerio_crawler.js

```diff
@@ -458,6 +494,34 @@ class CheerioCrawler {
             response.on('error', reject).pipe(parser);
         });
     }
+
+    /**
+     * Reads whole stream and concat buffer from it
+     * @param readStream
+     * @ignore
+     */
+    async _concatStreamToBuffer(readStream) {
```

Comment:
I'd move the `readStreamToString` from `http-request` to `apify-shared` and make it return a `Buffer` if `encoding` is `null`. Maybe a rename?

https://github.com/apifytech/http-request/blob/master/src/read_stream_to_string.js

Reply:
#471